core/src/main/java/org/opensearch/sql/calcite/CalciteRelNodeVisitor.java

```diff
@@ -1547,6 +1548,20 @@ private static void buildDedupNotNull(
     context.relBuilder.projectExcept(_row_number_dedup_);
   }
 
+  @Override
+  public RelNode visitMvExpand(MvExpand node, CalcitePlanContext context) {
```

Comment:
Is `limit` option the only difference between `expand` and `mvexpand`? I see `expand` is translated to SQL below. If so, we just need to add a `LIMIT` clause in inner query? Ref: https://github.com/opensearch-project/sql/blob/main/ppl/src/test/java/org/opensearch/sql/ppl/calcite/CalcitePPLExpandTest.java#L105

cc: @yuancu